bgpd/bgp_route.c

```diff
@@ -4438,8 +4438,12 @@ int bgp_nlri_parse_ip(struct peer *peer, struct attr *attr,
 					   BGP_ROUTE_NORMAL, NULL, NULL, 0,
 					   NULL);
 
-		/* Address family configuration mismatch or maximum-prefix count
-		   overflow. */
+		/* Do not send BGP notification twice when maximum-prefix count
+		 * overflow. */
```

Comment:
This comment isn't quite right, the opening and closing `/* */` need to be on their own lines.

Reply:
Hm, strange. Because it was "corrected" by .clang-format rules automatically.

Reply:
Yes, clang-format is not perfect. It does not have sufficient smarts to reflow comments to Linux style afaik.

In any case I'm not blocking the PR on this it would just be nice to fix.

Reply:
@qlyoung I'm just really curious why this cosmetic change is needed? maybe a good reason I don't know :)